### ISSUE TITLE
RDM Redoublement Combo broken after today's update

### ISSUE THREAD

**Issue by jtabox** [Grammernatzi/XIVComboPlugin#94]
Hey,
I just updated the plugin (to version 1.2.8.7 according to the plugin installer) and the update seems to have broken the Redoublement combo. Namely the option #190 seems to be always active: if I cast one of the two AoE spells (Veraero II and Verthunder II), Redoublement combo swaps automatically to Moulinet, even though I have the option deactivated in the settings.
Pretty sure this came with today's update, because I was playing RDM yesterday the whole day and didn't have this issue, it came up after I updated the plugin today.
Tried changing settings around, but it doesn't seem to matter. I'm lvl 90 RDM in case it matters.

**Comment by Grammernatzi** [comment]
Yes, it was ignoring the option and just doing it; that said, it shouldn't have been causing any issues? I designed it so it'd never replace Redoublement combo with Moulinet in a situation where you'd want to use the former. Regardless, I'm pushing an update because it was ignoring the option.

**Comment by jtabox** [comment]
Yes, it seems to work as intended with the last update. Thanks for replying+fixing so quickly :)
